src/main/java/wooteco/subway/path/domain/fare/Fare.java

```diff
@@ -0,0 +1,22 @@
+package wooteco.subway.path.domain.fare;
+
+import wooteco.subway.path.domain.fare.age.AgeStrategy;
+import wooteco.subway.path.domain.fare.distance.DistanceStrategy;
+
+public class Fare {
+    private static final int BASIC_FARE = 1250;
+    private final DistanceStrategy distanceStrategy;
```

Comment:
```suggestion
    private static final int BASIC_FARE = 1250;
    
    private final DistanceStrategy distanceStrategy;
```

클래스 변수와 인스턴스 변수 사이에 공백을 추가한다면 가독성이 상승합니다!